pipelines/documentation/build.yaml

```diff
@@ -65,12 +65,21 @@ steps:
     displayName: 'Copy testing output'
     workingDirectory: $(WorkingDir)
 
+<<<<<<< HEAD
```

Comment:
Looks like a merge-conflict artefact has made its way in…?

Reply:
Sorted :)